src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceMatchingIntegrationTest.kt

```diff
@@ -13,6 +13,24 @@ import java.time.LocalDate
 
 class LicenceMatchingIntegrationTest : IntegrationTestBase() {
 
+  @Test
+  @Sql(
+    "classpath:test_data/seed-a-few-licences.sql"
+  )
+  fun `Find approved licences`() {
```

Comment:
Improvements for each test in this suite: 

`Find approved licences`: 21 queries -> 5 queries (4n + 1 -> n + 1)
  
`Get licences matches - no filters`: 4 queries-> 1 query

`Get licences matches - by list of staff identifiers`:   8 queries -> 2 queries
  
  `Get licence matches - by list of staff identifiers and statuses`: 5 queries -> 2 queries

 
  `Get licence matches - by list of prisons and statuses`: 8 -> 2 queries
  
`Get licence matches - no matching filters`: 1 query  -> 1 query
  
 `Get licence matches - sort by conditional release date`: 8 queries -> 2 queries